src/clj/game/cards-agendas.clj

```diff
@@ -254,7 +254,8 @@
                        :effect (effect (corp-install target nil {:install-state :rezzed}))} card targets))}
 
    "Mandatory Upgrades"
-   {:effect (effect (gain :click 1 :click-per-turn 1))}
+   {:effect (effect (gain :click 1 :click-per-turn 1))
+    :leave-play (effect (lose :click-per-turn 1))}
```

Comment:
Spot on--thanks for the tip. I was also alerted that the Runner forfeiting a stolen Mandatory Upgrades will drop to 3 clicks per turn, so I've fixed that as well and changed `desactivate` to only do the `:leave-play` effect if the forfeited agenda came from the Corp's score area.